webapp/src/pages/Settings.tsx

```diff
@@ -41,6 +42,18 @@ const STEPPER: Record<string, InputBaseComponentProps> = {
 
 type SubConfigKeys = keyof PickByValue<AzimuthConfig, object>;
 
+const UNITS: Record<string, string> = {
+  max_delta_class_imbalance: "ms",
+  max_delta_mean_tokens: "ms",
+  max_delta_representation: "ms",
+  max_delta_std_tokens: "ms",
+  min_num_per_class: "ms",
+  short_sentence_max_token: "ms",
+  long_sentence_min_token: "ms",
+  conflicting_neighbors_threshold: "ms",
+  no_close_threshold: "ms",
```

Comment:
Is everything in millisecondes?

Is it a problem if `iterations` and `high_epistemic_threshold` are missing? Or should we exclude fields that have no units?

- For all the fields in number of `tokens`, it should be `"tokens"`, or nothing as it might already be clear. That would be similar for the `iterations` fields.
- `min_num_per_class` is in `"samples"`.
- The rest of the fields plus `high_epistemic_threshold`, the ones with `max: 1`, are ratios (like percentages) so they don't have units. Alternatively, we could consider displaying them in percentages (multiplying by 100 and showing "%"). What do you think @gabegma?
```suggestion
  max_delta_class_imbalance: "",
  max_delta_mean_tokens: "tokens",
  max_delta_representation: "",
  max_delta_std_tokens: "tokens",
  min_num_per_class: "samples",
  short_sentence_max_token: "tokens",
  long_sentence_min_token: "tokens",
  conflicting_neighbors_threshold: "",
  no_close_threshold: "",
```

Reply:
Your edits are perfect Joseph, and I like the idea of showing them as percentages!

Reply:
Please correct me if I am wrong. As per my understanding, we display the value as ratios with % unit for the following fields:
high_epistemic_threshold 
conflicting_neighbors_threshold 
no_close_threshold 
max_delta_class_imbalance 
max_delta_representation

Please also tell me what the units could be for - temperature and threshold.

Reply:
Yes, all those could be displayed as percentages, including temperature and threshold in postprocessors.

Reply:
Thank you @gabegma for the details and please tell me what could be the units for temperature and threshold.

Reply:
I would say no unit!

Reply:
Ho right, good catch for the `temperature`!
For the confidence threshold however, we display it as a percentage everywhere else in our app. Shouldn't we be consistent here?
![image](https://user-images.githubusercontent.com/8386369/212164501-0123a2c9-3db4-4bf9-8ecb-170b224a7c2a.png)
![image](https://user-images.githubusercontent.com/8386369/212164286-e8b0a457-8cf0-4f9b-ba37-6c6050eecaed.png)
![image](https://user-images.githubusercontent.com/8386369/212164541-7c0f59de-fe34-4cbb-8ad8-d579ecee0940.png)

Reply:
Ok - we can display it as a percentage.